src/components/button/button.tsx

```diff
@@ -8,17 +8,22 @@ interface IButton {
 
 // Default Button Styles
 const Button = styled.button<IButton>`
-  height: 3rem;
-  min-width: 10rem;
-  width: ${props => (props.screen === "small" ? "70%" : "15rem")};
+  height: 4.5rem;
+  min-width: 8rem;
+  width: ${props => (props.screen === "small" ? "50%" : "15rem")};
```

Comment:
By changing the width to 50% on small screens you are going against the design which almost takes 80% of the whole screen width.

Reply:
```suggestion
  width: ${props => (props.screen === "small" ? "60%" : "15rem")};
```
![image](https://user-images.githubusercontent.com/46858011/89512851-6cc28900-d7f1-11ea-81cc-ee4dfe5f5099.png)

Actually see this. In small screens, it seems no way 70% of the width. That's why changed it to 50%, but this can be 55% or 60% as well to make it perfect.

Reply:
I finally made it 60% instead 70% after testing out different values. Resolving this thread.